src/Workspaces/Core/Portable/Workspace/Solution/SolutionCompilationState.cs

```diff
@@ -1140,32 +1206,33 @@ private SolutionCompilationState ComputeFrozenSnapshot(CancellationToken cancell
         }
 
         static SolutionCompilationState ComputeFrozenPartialState(
```

Comment:
Consider at this point just inlining this into the caller at this point.

Reply:
i like it extracted sinec hte logic is non-trivial in some cases.